### ISSUE TITLE
Cannot access uielements inside (within) UITableView by using Appium

### ISSUE THREAD

**Issue by igalep** [appium/appium#5675]
I've being trying to access to the uielements (labels) which are located within UITableViewCell which is located in UITableView. (see the picture below)

![screen shot 2015-09-30 at 10 40 39 am](https://cloud.githubusercontent.com/assets/3533224/10188254/8f3184b0-6766-11e5-995f-28970745df31.png)

I set a setIsAccessibilityElement:YES and setAccessibilityLabelto each required label I can access with no issues.

I'm adding an accessibility to the cell , still can access to each label in addition to the cell as well

I'm adding accessibility to the table from this point Appium cannot recognise/inspect the cell and the labels , but only the table ...

<img width="880" alt="screen shot 2015-09-30 at 10 54 11 am" src="https://cloud.githubusercontent.com/assets/3533224/10188265/b0f468ec-6766-11e5-8da0-f1b0fdbcd903.png">

The thing is , IOS Accessibility inspector (runs from IOS simulator) does able to recognise the table / cell / labels

Any idea what Do I need in the sourcecode of the app in order to make it work with Appium ?
Thanks

**Comment by v0l0d** [comment]
I have also that issue
Looks like it depends on table cell elements
I'm ready to provide iOS project source code and the APP for the appium to run in the simulator

**Comment by SrinivasanTarget** [comment]
Please try a latest version of appium and raise a new issue along with logs incase if you hit the same.

**Comment by SrinivasanTarget** [comment]
@triager please close
